### PR TITLE
bsp: Make the bsp layer buildable on Fedora 25

### LINKED ISSUE #42: Building the image on Fedora 25 fails due to missing acpica-native
Hi!

I'm trying to build the image on a Fedora 25 system.
The build fails with the following error message:

```
$ bitbake core-image-minimal
WARNING: Host distribution "Fedora-25" has not been validated with this version of the build system; you may possibly experience unexpected failures. It is recommended that you use a tested distribution.
Loading cache: 100% |###################################################################################################| Time: 0:00:00
Loaded 1343 entries from dependency cache.
Parsing recipes: 100% |#################################################################################################| Time: 0:00:00
Parsing of 888 .bb files complete (886 cached, 2 parsed). 1344 targets, 56 skipped, 0 masked, 0 errors.
NOTE: Resolving any missing task queue dependencies
ERROR: Nothing PROVIDES 'acpica-native' (but /home/build/poky/meta-iot2000/meta-iot2000-bsp/recipes-bsp/acpi-upgrades/acpi-upgrades.bb DEPENDS on or otherwise requires it). Close matches:
  cpio-native
  byacc-native
  asciidoc-native
ERROR: Required build target 'core-image-minimal' has no buildable providers.
Missing or unbuildable dependency chain was: ['core-image-minimal', 'core-image-minimal-initramfs', 'acpi-upgrades', 'acpica-native']
```

acpica-native is required by `meta-iot2000/meta-iot2000-bsp/recipes-bsp/acpi-upgrades/acpi-upgrades.bb` 

Who should provide acpica-native?
Is there something missing on my build system?

### PR DESCRIPTION
**Pull request by fbezdeka**
It's a non-breaking pull request because of doc fixes only.
I tested the build locally. No problems were detected. 

Please check the order inside `BBLAYERS`.

Closes #42.

### CODE REVIEW

File: meta-iot2000-bsp/README.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ Building the meta-iot2000 BSP Layer
 ===================================
 
 This uses Yocto 2.2 (Morty) with the 4.4 Linux kernel provided by Yocto and
-the meta-intel layer. The build has been tested successfully on Debian 8 and
-OpenSuse 42.2, but other recent distros are expected to work as well.
+the meta-intel layer. The build has successfully been tested on Debian 8,
```
Comment (jan-kiszka) [inline meta-iot2000-bsp/README.md:26]: Are you sure about this reordering of words? I'm not a native speaker, just unsure.

File: meta-iot2000-bsp/README.md [hunk 2]
```diff
@@ -67,6 +69,7 @@ $ source poky/oe-init-build-env iot2000-build
    /home/build/poky/meta \
```
Comment (jan-kiszka) [inline meta-iot2000-bsp/README.md:69]: Makes "+9,7" in diff logic, nitpicking...

**Review by jan-kiszka — COMMENTED** [review]
Thanks for fixing! Just one question and a minor remark.

File: meta-iot2000-bsp/README.md (hunk 1 above)
Comment (fbezdeka) [reply meta-iot2000-bsp/README.md:26]: Yes. (But no native speaker as well...)

File: meta-iot2000-bsp/README.md (hunk 2 above)
Comment (fbezdeka) [reply meta-iot2000-bsp/README.md:69]: Good catch!

**Comment by fbezdeka** [comment]
I can squash commits, but I'm unsure if i have to open a new pull-request then.

**Comment by jan-kiszka** [comment]
Just force-push to the same branch again. This PR will track that as update and open a second round.

**Comment by jan-kiszka** [comment]
Thanks, merged.

Please sign off your commits in the future, according to https://developercertificate.org.

**Pull request closed without merging**